.travis.yml

```diff
@@ -3,13 +3,26 @@ language: python
 sudo: false
 python:
     - 2.7
-    - 3.3
-    - 3.4
     - 3.5
+    - 3.6
+    - pypy
+    - pypy3
+matrix:
+    include:
+        - python: "3.7"
+          dist: xenial
+          sudo: true
 install:
+    - pip install coveralls
     - python bootstrap.py
     - bin/buildout
 script:
     - bin/test
```

Comment:
Does this collect coverage?  AFAICT bin/test is py.test, and I don't see where coverage tracking would get enabled by default.  The tox env also uses py.test and it enables coverage explicitly.

Reply:
No, it does not. 😢